apps/api/.env.example

```
@@ -29,6 +29,9 @@ GREDICE_DELIVERY_NOTIFICATION_ROLLOUT_STARTED_AT=
 # Customer delivery channel controls are always available in Garden.
 GREDICE_DELIVERY_NOTIFICATIONS_ENABLED=false
 GREDICE_BILLING_AUTOMATION_ENABLED=false
+# Independent server-authority gate for Garden Building activation and writes.
+# Saved structures must stay readable if this rollback gate is disabled.
+GREDICE_GARDEN_BUILDING_SYSTEM_ENABLED=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GREDICE_GARDEN_BUILDING_SYSTEM_ENABLED key should go before the GREDICE_NOTIFICATIONS_BULK_CAMPAIGNS_ENABLED key

(UnorderedKey)